middleware/logmiddleware.go

```diff
@@ -93,19 +94,33 @@ func logBeforeHandler(ctx loggingContext) {
 	}).Trace("incoming request")
 }
 
-func logAfterHandler(ctx loggingContext, startTime time.Time) {
-	glogger.Get(ctx.Context()).WithFields(logrus.Fields{
+func logAfterHandler(ctx loggingContext, startTime time.Time, err error) {
+	logger := glogger.Get(ctx.Context())
+	res := &Response{
+		StatusCode: ctx.Response().StatusCode(),
+		Body: map[string]interface{}{
+			"bytes": ctx.Response().BodySize(),
+		},
+	}
+
+	if err != nil {
+		if fiberErr, ok := err.(*fiber.Error); ok {
+			res = &Response{
+				StatusCode: fiberErr.Code,
+				Body: map[string]interface{}{
+					"bytes": len(fiberErr.Error()),
+				},
+			}
+		}
+	}
```

Comment:
This two could be unified into: `err != nil && ok`, because the type cast operator is safe on `nil` values.

Reply:
done!